docker/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.1'
+services:
+  mongodb:
+    image: mongo:latest
+    ports:
+      - 27017:27017
+    volumes:
+      - /Users/danielmasegosa/Workspaces/vgsma/docker/volumes/mongodb/data:/data/db
```

Comment:
I think this link should be changed to a more relative one

Reply:
You're right, I will change it

Reply:
I update the volume paths to relative